docs/provider.md

```diff
@@ -1,158 +1,294 @@
-| `--verbose` / `--no-verbose`               | Toggle verbose logging, defaults to False.                                                                                                                             |
-| `--log-dir TEXT`                           | The directory for the pact.log file.                                                                                                                                   |
-| `--log-level TEXT`                         | The logging level.                                                                                                                                                     |
-| `--enable-pending` / `--no-enable-pending` | Allow pacts which are in pending state to be verified without causing the overall task to fail. For more information, see [`pact.io/pending`](https://pact.io/pending) |
-| `--include-wip-pacts-since TEXT`           | Automatically include the pending pacts in the verification step. For more information, see [WIP pacts](https://docs.pact.io/pact_broker/advanced_topics/wip_pacts/)   |
-| `--help`                                   | Show this message and exit.                                                                                                                                            |
+More information on the selector options is available in the [API reference][pact.verifier.BrokerSelectorBuilder].
 
-<!-- markdownlint-disable code-block-style -->
+### Publishing Results
 
-??? note "Deprecated Options"
+To publish verification results to the Broker:
 
-    | Option                       | Description                                                                                                                                                                        |
-    | ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-    | `--pact-url TEXT`            | specify pacts as arguments instead. The URI of the pact to verify. Can be an HTTP URI, a local file or directory path. It can be specified multiple times to verify several pacts. |
-    | `--pact-urls TEXT`           | specify pacts as arguments instead. The URI(s) of the pact to verify. Can be an HTTP URI(s) or local file path(s). Provide multiple URI separated by a comma.                      |
-    | `--provider-states-url TEXT` | URL to fetch the provider states for the given provider API.                                                                                                                       |
+```python
+verifier = (
+    Verifier("my-provider")
+    .add_transport(url="http://localhost:8080")
+    .broker_source("https://my-broker.example.com", token="bearer-token")
+    .set_publish_options(
```

Comment:
how does one enable conditional publishing? such that they can run this test on their local machine and not publish results, but only publish from CI.

Is there a publishVerificationResult option, which the user can setup to be controlled by checking the env.CI flag?